main.tex

```diff
@@ -16,6 +16,7 @@
 \usepackage{lipsum}
 \usepackage{tabularx}
 \usepackage{graphicx}
+\usepackage{float}
```

Comment:
It looks like the float package is not really required here. Why did you include that?

Reply:
Because otherwise Latex would place one of the figures in the completely wrong place.
Specifically for this
\begin{figure}[H]
...
\end{figure}

Reply:
Do you actually need the package? We used `\begin{figure}[H]` without explicitly including that package and it seemed to work.